src/components/CAUField.jsx

```diff
@@ -33,7 +33,7 @@ export function CAUField(props) {
     const collective_installation = props.data.collective_installation
     const cups = props.data.cups.slice(0, compact.length)
     if (collective_installation === false) {
-      const cau_cups = compact.slice(0, compact.length)      
+      const cau_cups = compact.slice(0, compact.length > 20 ? 20 : compact.length)      
```

Comment:
I am not sure on whether using Math.min() here is worth for simplicity, intent declaration and debugging paranoia. In python i would. In JS Math namespaces makes me wonder.